WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -403,6 +406,28 @@ private extension QuickStartTourGuide {
         return countChecklistCompleted(in: list, for: blog) >= list.count
     }
 
+    /// Returns a list of all available tours that have not yet been completed
+    /// - Parameter blog: blog to check
+    func uncompletedTours(for blog: Blog) -> [QuickStartTour] {
+        let completedTours: [QuickStartTourState] = blog.completedQuickStartTours ?? []
+        let allTours = QuickStartFactory.allTours(for: blog)
+        let completedIDs = completedTours.map { $0.tourID }
+        let uncompletedTours = allTours.filter { !completedIDs.contains($0.key) }
+        return uncompletedTours
+    }
+
+    /// Check if the provided tour have not yet been completed and is available to complete
+    /// - Parameters:
+    ///   - tour: tour to check
+    ///   - blog: blog to check
+    /// - Returns: boolean, true if the tour is not completed and is available. False otherwise
+    func isTourNotCompleted(tour: QuickStartTour, for blog: Blog) -> Bool {
```

Comment:
[nit] Wdyt of inverting the method name to `isTourCompleted`? I think the double negative for `!isTourNotCompleted` can be a bit confusing

Reply:
Renamed it to `isTourAvailableToComplete` to avoid confusion and to avoid changing it's definition 👍 
Done in b3b6bc188fee7997cd1d0f83a8bd22c2a6a91d96